sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Diagnostics/EventDataInstrumentation.cs

```diff
@@ -14,7 +14,7 @@ namespace Azure.Messaging.EventHubs.Diagnostics
     internal static class EventDataInstrumentation
     {
         /// <summary>The client diagnostics instance responsible for managing scope.</summary>
-        public static ClientDiagnostics ClientDiagnostics { get; } = new ClientDiagnostics("Azure.Messaging.EventHubs", true);
+        public static ClientDiagnostics ClientDiagnostics { get; } = new ClientDiagnostics("Azure.Messaging.EventHubs", "Microsoft.EventHub", true);
```

Comment:
Should we consider adding the attribute to the two Event Hubs packages just to keep them aligned with the other SDKs?

Reply:
Attribute is required when an instance of client options is used to create ClientDiagnostics which we don't do in EH and use a static instead. I wanted to avoid large changes to libraries as part of this.